connect-example/src/main/java/com/stripe/android/connect/example/data/SettingsService.kt

```diff
@@ -0,0 +1,137 @@
+package com.stripe.android.connect.example.data
+
+import android.app.Application
+import android.content.Context
+import android.content.SharedPreferences
+import androidx.core.content.edit
+import com.stripe.android.connect.example.ui.appearance.AppearanceInfo
+
+class SettingsService private constructor(context: Context) {
+
+    private val sharedPreferences: SharedPreferences by lazy {
+        context.getSharedPreferences("SettingsService", Context.MODE_PRIVATE)
+    }
+
+    fun getSelectedServerBaseURL(): String? {
+        return sharedPreferences.getString(SERVER_BASE_URL_KEY, null)
+    }
+
+    fun setSelectedServerBaseURL(value: String) {
+        sharedPreferences.edit { putString(SERVER_BASE_URL_KEY, value) }
+    }
+
+    fun getAppearanceId(): AppearanceInfo.AppearanceId? {
+        val appearanceId = sharedPreferences.getString(APPEARANCE_ID_KEY, null) ?: return null
+        return AppearanceInfo.AppearanceId.valueOf(appearanceId)
```

Comment:
you might want to do a try catch here and return null just in case things get renamed in the future

Reply:
I didn't think about this, good idea!

Reply:
I ended up doing firstOrNull instead for the easier null safety